policy/pipeline/required_tasks_test.rego

```diff
@@ -160,17 +244,45 @@ _missing_tasks_violation(tasks) = errors if {
 	}
 }
 
+_missing_pipeline_tasks_violation(tasks) = errors if {
```

Comment:
This seems [not to be covered by tests](https://app.codecov.io/github/hacbs-contract/ec-policies/commit/0ddc7d4b1460b99e71cb3bad672e06e643a8138c#4a4584e63a2e97d4b926b350fb8694bd-L247) and I can't find where it is used, so can it be removed?